.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2 # use CircleCI 2.0
 jobs: # a collection of steps
   build: # runs not using Workflows must have a `build` job as entry point
     docker: # run the steps with Docker
-      - image: circleci/node:8.8.1 # ...with this image as the primary container; this is where all `steps` will run
+      - image: circleci/node:12.16.0 # ...with this image as the primary container; this is where all `steps` will run
```

Comment:
May I ask the reason for changing the version here?

Reply:
Mostly because node 8 is no longer supported https://blog.risingstack.com/update-nodejs-8-end-of-life-no-support/

I believe that this change is already in master though?

Reply:
You're right. :-)